requirements.txt

```diff
@@ -0,0 +1,9 @@
+numpy~=1.15.0
```

Comment:
Is there a reason that you use "~=" instead of ">="? The same for periodictable and scipy. (Sorry for the picky comments.)

Reply:
Sure thing: it's because numpy and scipy use semantic versioning and it's unlikely we will work with a major break before testing it.

Since I want to avoid being ping-ed by users on Slack because they got the two-hour-old numpy release and nothing works anymore, I rather like the pip dependency resolver  to load the last 1.* release of numpy for instance and update at our own speed in the next monthly release.

Reply:
Ok, being conservative about releases is probably not a bad thing. Though it'll probably be a long time before we'll have NumPy version 2.

Reply:
Yes, that's why I think it's quite save (I also try to avoid to over-constrain those)